src/Form/FormAnalyzer.js

```diff
@@ -256,10 +263,7 @@ class FormAnalyzer {
             this.updateSignal({string, strength, signalType: `external link: ${string}`, shouldFlip})
         } else {
             // any other case
-            // only consider the el if it's a small text to avoid noisy disclaimers
-            if (removeExcessWhitespace(el.textContent)?.length < constants.TEXT_LENGTH_CUTOFF) {
```

Comment:
The length check has been moved to `updateSignal` so we're sure all these regexes are safeguarded against extra long text.